tests/wpt/tests/content-security-policy/sandbox/autoplay-disabled-by-csp.html.headers

```diff
@@ -1 +0,0 @@
-Content-Security-Policy: sandbox allow-forms
```

Comment:
This test would have needed `allow-scripts` to run. However, I then realized that the form doesn't run when `allow-scripts` is set: https://github.com/servo/rust-content-security-policy/pull/16/files#diff-b93597bfc0a91f61fffed5b776df0e5021061d0c47bbb61d7eb0e043962fa523L50 Since we weren't checking for `allow-scripts`, this wasn't caught before.